src/index.js

```diff
@@ -0,0 +1,23 @@
+import express from "express";
+import cookieParser from "cookie-parser";
```

Comment:
this is not been used, please remove it.

Reply:
I have removed it.